test/cloud_testing/platforms/ubuntu_x86_64_test.sh

```diff
@@ -33,7 +33,7 @@ CVMFS_TEST_CLASS_NAME=ServerIntegrationTests                                  \
                                  --                                           \
                                  src/5*                                       \
                                  src/6*                                       \
-                                 src/7*                                       \
+#                                 src/7*                                       \
```

Comment:
I think this wouldn't work: the final line break is ignored because the line is a comment.